### DISCUSSION TITLE
Products sync

### DISCUSSION THREAD

**Discussion by serpico** [openfoodfacts/open-prices#460] [General]
[Raphael] Every afternoon we run a script to sync the OFF database (food, product, beauty, pet food + obsolete) with OP products.

### Code

https://github.com/openfoodfacts/open-prices/blob/main/open_prices/common/tasks.py

### Stats

|Date|OFF + obsolete|OPF|OBF|OPFF|_unknown_|
|---|---|---|---|---|---|
|2024-12-04|3624143|15741|31463|9955|5484|

### Related issues

- https://github.com/openfoodfacts/open-prices/issues/114
- https://github.com/openfoodfacts/open-prices/issues/281
- https://github.com/openfoodfacts/open-prices/issues/243 & https://github.com/openfoodfacts/open-prices/issues/243 & https://github.com/openfoodfacts/open-prices/issues/465 & https://github.com/openfoodfacts/open-prices/issues/551
- https://github.com/openfoodfacts/open-prices/issues/620

**Comment by raphodn** [comment]
## Sync issues (since migrating to Django)

> I thought it was related to #407 ( feel free to close this is it's the case ) so I didn't raise this issue before, but it seems the products weren't synced for a while ( I saw 04/09/24 ). Symptom is similar to #315

indeed @serpico you are right, I'm facing some issues with the OxF product sync.
Since we migrated to Django, it's not working correctly.
The culprit is #376

I tried to do some corrections, last one a few days ago, but still no success..

**Reply by raphodn** [reply]
fyi I ran the OPF sync a few times manually, but not the others

**Reply by raphodn** [reply]
I created an issue to track this : https://github.com/openfoodfacts/open-prices/issues/465

**Reply by raphodn** [reply]
also renaming the content in case someone stumbles on this :)

**Reply by raphodn** [reply]
fixed :)

**Comment by raphodn** [comment]
The OPF sync seems not to work anymore. Maybe due to the changes happening the openfoodfacts-server (and migrating all the flavors into one place)

**Reply by raphodn** [reply]
Opened an issue in Product Opener: https://github.com/openfoodfacts/openfoodfacts-server/issues/11118

**Comment by raphodn** [comment]
@raphael0202 added a live sync with OFF's product changes :tada: (https://github.com/openfoodfacts/open-prices/issues/620)

Haven't tested yet !
In any case, with the current frontend, you need to refresh the page you're on to see the changes on the product card :)
